curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-functional-programming-by-building-a-spreadsheet/64347464f78cd9209545f35c.md

```diff
@@ -22,7 +22,7 @@ assert.match(code, /letters\.forEach\(/);
 You should pass your `createLabel` function reference to the `.forEach()` method.
 
 ```js
-assert.match(code, /letters\.forEach\(\s*createLabel\s*\)/);
+assert.match(code, /letters\.forEach\(\s*(\(\s*letter\s*\)|\s*letter)\s*=>\s*createLabel\(\s*letter\s*\)\s*\)|letters\.forEach\(\s*createLabel\s*\)/);
```

Comment:
```suggestion
assert.match(code, /letters\.forEach\(\s*(?:\(\s*([\S]*)\s*\)|\s*([\S]*))\s*=>\s*createLabel\(\s*\1\2\s*\)\s*\)|letters\.forEach\(\s*createLabel\s*\)/);
```
I played with it a bit, this version gives ability to use any parameter name (since it's not defined by instructions).

Reply:
works like a charm